util/src/main/java/org/datacommons/util/StatChecker.java

```diff
@@ -471,4 +451,17 @@ private boolean checkSvObsValueInconsistency(McfGraph.PropertyValues node) {
       return true;
     }
   }
+
+  private synchronized void extractStatVarInfoFromNode(McfGraph.PropertyValues node) {
+    String svDcid = McfUtil.getPropVal(node, Vocabulary.VARIABLE_MEASURED);
```

Comment:
From what I recall, this bubbles up in the profile, and we're looking up every prop thrice now:
1. here
2. checkSvObsInGraph
3. extractSeriesFromNode

The first two happen for every svobs too. Perhaps we could extract into a struct and pass it around? If you agree, maybe add TODO for future PR.

Reply:
done